ReText/window.py

```diff
@@ -308,44 +325,146 @@ def __init__(self, parent=None):
         self.actionAbout.setMenuRole(QAction.MenuRole.AboutRole)
         self.actionAboutQt = self.act(self.tr('About Qt'))
         self.actionAboutQt.setMenuRole(QAction.MenuRole.AboutQtRole)
-        self.actionAboutQt.triggered.connect(qApp.aboutQt)
-        availableMarkups = markups.get_available_markups()
-        if not availableMarkups:
-            print('Warning: no markups are available!')
-        if len(availableMarkups) > 1:
-            self.chooseGroup = QActionGroup(self)
-            markupActions = []
-            for markup in availableMarkups:
-                markupAction = self.act(markup.name, trigbool=self.markupFunction(markup))
-                if markup.name == globalSettings.defaultMarkup:
-                    markupAction.setChecked(True)
-                self.chooseGroup.addAction(markupAction)
-                markupActions.append(markupAction)
+        self.actionAboutQt.triggered.connect(QApplication.instance().aboutQt)
+
         self.actionBold = self.act(self.tr('Bold'), shct=QKeySequence.StandardKey.Bold,
-            trig=lambda: self.insertFormatting('bold'))
+            trig=self._on_formatting_action)
         self.actionItalic = self.act(self.tr('Italic'), shct=QKeySequence.StandardKey.Italic,
-            trig=lambda: self.insertFormatting('italic'))
+            trig=self._on_formatting_action)
         self.actionUnderline = self.act(self.tr('Underline'), shct=QKeySequence.StandardKey.Underline,
-            trig=lambda: self.insertFormatting('underline'))
+            trig=self._on_formatting_action)
+
+    def _create_menus(self):
+        self._create_menu_file()
+        self._create_menu_edit()
+        self._create_menu_help()
```

Comment:
Similarly, maybe better to name `createFileMenu`, `createEditMenu`, `createHelpMenu`, etc?

By the way, this file uses `camelCase` for consistency with Qt. Also, there is no need for leading underscore, since effectively everything is private.

Also, I would be extra grateful if you submit this refactoring as a separate PR, this way I can merge it sooner.